packages/fuels-programs/src/calls.rs

```diff
@@ -7,16 +7,45 @@ pub mod utils;
 
 pub use call_handler::*;
 pub use contract_call::*;
+use fuel_types::BlockHeight;
 pub use script_call::*;
 
 /// Used to control simulations/dry-runs
 #[derive(Debug, Clone, Default)]
```

Comment:
Perhaps it would be easier to reason about what is the default if we separate out the default impl to call `Execution::realistic`. 

This way we have to know the default behavior of all of the fields of the struct.

Reply:
https://github.com/FuelLabs/fuels-rs/pull/1628/commits/7c97c96bc50f607b12dfaa81c21a98e3bb237d5d